src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -101,6 +101,7 @@ internal static class EnforceOnBuildValues
         public const EnforceOnBuild ConsecutiveBracePlacement = /*IDE2002*/ EnforceOnBuild.WhenExplicitlyEnabled;
         public const EnforceOnBuild ConsecutiveStatementPlacement = /*IDE2003*/ EnforceOnBuild.WhenExplicitlyEnabled;
         public const EnforceOnBuild ConstructorInitializerPlacement = /*IDE2004*/ EnforceOnBuild.WhenExplicitlyEnabled;
+        public const EnforceOnBuild ForEachCast = /*IDE0210*/ EnforceOnBuild.WhenExplicitlyEnabled;
```

Comment:
nit: consider keeping the ascending order, ie, place the new entry between IDE2000 and IDE0150

Reply:
thanks.  done